dm/dm-best-practices.md

```diff
@@ -0,0 +1,233 @@
+---
+title: TiDB DM Best Practices
```

Comment:
It is recommended  to change to TiDB Data Migration (DM) Best Practices

Reply:
Updated.